compute_tools/src/compute.rs

```diff
@@ -2207,6 +2208,58 @@ pub fn forward_termination_signal() {
         let ss_pid = nix::unistd::Pid::from_raw(ss_pid as i32);
         kill(ss_pid, Signal::SIGTERM).ok();
     }
+
+    // Terminate pgbouncer with SIGKILL
+    match pid_file::read("/etc/pgbouncer/pid".into()) {
```

Comment:
Would love to see the path pulled into a constant in a `pgbouncer` module (`compute_tools/src/pgbouncer.rs`).